kythe/go/extractors/config/runextractor/compdb/compdb.go

```diff
@@ -39,10 +39,25 @@ import (
 // A compileCommand holds the decoded arguments of a LLVM compilation database
 // JSON command spec.
 type compileCommand struct {
+	Arguments []string
 	Command   string
 	Directory string
 }
 
+func (cc *compileCommand) asCommand() string {
+	if len(cc.Arguments) > 0 {
+		return shell.Join(cc.Arguments)
+	}
+	return cc.Command
+}
+
+func (cc *compileCommand) asArguments() (args []string, ok bool) {
```

Comment:
The return values are obvious from context without the names and should likely be omitted.

Reply:
Thanks for reviewing, removed the names.